source/PlayerInfo.cpp

```diff
@@ -943,7 +943,7 @@ void PlayerInfo::SellShip(const Ship *selected)
 			ForgetShip(*it->get());
 			ships.erase(it);
 			flagship.reset();
-			return;
+			break;
```

Comment:
Oh, it was in a for loop. ~~return was fine, then~~

Reply:
You sure? Because really I think that since the end of the function is right below this, its clearer to do a break instead of return